chevah/brink
# [#4166] Fix paver lint quick.

Scope
=====

This fix paver lint when running in quick mode.

Changes
=======

A newer scame project is used. The git diff code was moved to scame.

The `lint` paver task is a self contained.

the main changes are in the scame project

How to try and test the changes
===============================

reviewers: @brunogola 

check that changes make sense.

the new scame and brink packages were already released and used in my http timeout branch on server
